pkgs/by-name/is/isd/package.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  python3,
```

```suggestion
  python3Packages,
```

For better cross compilation support.